ko/libraries/index.md

```diff
@@ -35,15 +35,15 @@ list`를 사용합니다. 좀 더 정보가 필요하시면 아래 내용을 좀
 소스코드는 GitHub에서 볼 수 있습니다.
 
 [**The Ruby Toolbox**][6]는 오픈소스이며 Ruby 프로젝트를 검색하기 쉽게 하기 위한
-프로젝트입니다. 다양한 종류의 일반적인 개발 작업을 카테고리로 릴리스나 커밋의
-활발함, 라이브러리의 의존관계 등 각 프로젝트의 다양한 정보를 모았습니다. 또 RubyGems.org와
-GitHub의 인기를 바탕으로 프로젝트의 순위를 매깁니다. 검색을 하면 원하는 것을 쉽게 찾으실
-수 있을 것입니다.
+프로젝트입니다. 다양한 개발 작업을 위한 카테고리를 가지고 있으며,
+RubyGems.org와 GitHub에서의 인기도를 기반으로 프로젝트를 평가합니다.
+또한, 릴리스 및 커밋 활동 혹은 의존성과 같이 프로젝트에 관한 많은 정보를 수집합니다.
```

Comment:
> It has categories for various common development tasks,
collects a lot of information about the projects like release and commit
activity or dependencies and rates projects based on their popularity on
RubyGems.org and GitHub.

Maybe we can follow the original order of the sentence.

```suggestion
프로젝트입니다. 다양한 개발 작업을 위한 카테고리를 가지고 있으며,
릴리스, 커밋 활동 혹은 의존성과 같이 프로젝트에 관한 많은 정보를 수집합니다.
또한 RubyGems.org와 GitHub에서의 인기도를 기반으로 프로젝트를 평가합니다.
```

Reply:
I've made the changes as suggested, following the original order of the sentence. Thank you for the helpful review.